templates/base.html

```diff
@@ -22,6 +22,10 @@
 <title>{% block page_title %}{% endblock %}{{ APP_TITLE }}</title>
 <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
 
+<link rel="manifest" href="/static/manifest.json">
+
+<meta name="theme-color" content="#standalone">
```

Comment:
How about adding `<link rel="icon" sizes="192x192" href="/static/img/chrome_logo_192.png">` as well as mentioned in https://developers.google.com/web/updates/2014/11/Support-for-theme-color-in-Chrome-39-for-Android?